tests/jobscript/05-global-config.t

```diff
@@ -17,6 +17,9 @@
 #-------------------------------------------------------------------------------
 # Test that global config is used search for poll
 . "$(dirname "${0}")/test_header"
+
+# TODO - replace this with a Cylc 8 test
+skip_all "This test is broken pending full implementation of platforms"
```

Comment:
We are not intending to have this sort of globalconfig work in Cylc 8. The best this test should give is a helpful warning, but that warning should already have been given elsewhere. More likely we need to re-write this in a platformsy way.

See issue https://github.com/cylc/cylc-flow/issues/3481